src/fusion.cpp

```diff
@@ -108,6 +108,50 @@ miopenStatus_t ConvBiasActivFusion(Handle& handle,
     return miopenStatusSuccess;
 }
 
+void AllocateConvBiasActivFusionInvokerBuffer(const FusionContext& context,
+                                              const FusionDescription& problem,
+                                              miopen::OperatorArgs& params,
+                                              AnyInvokeParams& fused_invoker)
```

Comment:
```suggestion
std::tuple<miopen::OperatorArgs, AnyInvokeParams> AllocateConvBiasActivFusionInvokerBuffer(const FusionContext& context,
                                              const FusionDescription& problem)
```

Sidenote: AnyInvokeParams is not an invoker